src/modules/corp-type-module/corp-type-module.ts

```diff
@@ -42,6 +42,7 @@ export enum CorpTypeCd {
   SOCIETY = 'S',
   SOCIETY_BRANCH = 'SB',
   SOLE_PROP = 'SP',
+  NR_SOLE_PROP = 'FR', // SPECIAL NAMEREQUEST-ONLY ENTITY TYPE
```

Comment:
1. Please keep this list sorted by code (see comment on line 7).
2. This new code needs an object in the CorpInfoArray below (for lookup purposes).

Reply:
Updated!

Reply:
Object looks good.

I believe sort is by code not key :)

Reply:
🤦  Sorry, will fix

Reply:
updated, only took me 12 commits and 3 cups of coffee to get them alphabetical. It must be Friday